day2/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,18 @@
+function solution(numbers) {
+    let sum = 0;
+    let answer = [];
```

Comment:
```suggestion
    let answer = [];
```
![image](https://github.com/hobiJeong/Jbhcdstory/assets/83463918/637d12c3-1dbb-45c6-8805-5461e00238fb)

https://nenara.com/75